perfmetrics/scripts/gsheet/gsheet.py

```diff
@@ -12,6 +12,7 @@ def _get_sheets_service_client():
   service = build('sheets', 'v4', credentials=creds)
   return service
 
```

Comment:
Can you write a comment here about how this file works?

Reply:
done